test/export/test_converter.py

```diff
@@ -605,12 +652,11 @@ def forward(self, x: torch.Tensor):
         # Nested module testing.
         inp = (torch.ones(3),)
         orig_m = NestedM(3)
-        # TODO: fix trace: state_dict is not equal.
-        ep_list = self._check_equal_ts_ep_converter(orig_m, inp, ["script"])
+        ep_list = self._check_equal_ts_ep_converter(orig_m, inp)
 
         t = inp[0]
         t -= 0.8
-        for ep in ep_list:
+        for ep in ep_list[1:]:
```

Comment:
can we leave a comment here that the jit traced module just goes through control flow?